xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/Translations.ru.xml

```diff
@@ -64,6 +64,17 @@ platform.wiki.users.leave.confirmation=Вы уверены, что хотите
 platform.wiki.users.leave.confirmation.no=Нет
 platform.wiki.users.leave.confirmation.yes=Да
 platform.wiki.users.leave.empty=Покинуть Вики
-platform.wiki.users.leave.title=Покинуть Вики "{0}" 
+platform.wiki.users.leave.title=Покинуть Вики "{0}"
+## Create
```

Comment:
@atallahade : any idea why we get this comment?

Reply:
It might actually be Weblate taking the comment from the source. Got to check

Reply:
Yes looks like comments coming from the source.